src/Inciter/AMR/edge.cpp

```diff
@@ -0,0 +1,9 @@
+#include "edge.h"
+
+// Implementation of the friend function, extending osstream and not an edge_t
+// member
+std::ostream& operator<<(std::ostream& os, const edge_t& e)
+{
+    os << e.data.first << "-" << e.data.second;
+    return os;
+}
```

Comment:
You mean C (as C and not C++) and CPP (as really C++)? If so, I thought for C one would use `.c`, i.e. lower case.